src/main/java/LittleHarppy/LittleHarppy.java

```diff
@@ -0,0 +1,55 @@
+package LittleHarppy;
+
+import file.Storage;
+import java.io.*;
```

Comment:
Avoid wildcard imports, that is against the java coding standard we follow in class. Consider importing only th classes you use in the program.